dev/ci/ci-basic-overlay.sh

```diff
@@ -104,11 +104,6 @@ project coqhammer "https://github.com/lukaszcz/coqhammer" "master"
 ########################################################################
 project geocoq "https://github.com/GeoCoq/GeoCoq" "master"
 
-########################################################################
-# Flocq 3
-########################################################################
-project flocq3 "https://gitlab.inria.fr/flocq/flocq" "flocq-3"
-
 ########################################################################
 # Flocq (master)
```

Comment:
Maybe the parenthesis is not necessary anymore.

Reply:
Done.